### ISSUE TITLE
Fix slide transitions to work in Opera

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#222]
```
The CSS you've used for the slide transitions goes against the current 
spec--shorthand margin and padding aren't transformable:

See 7.1 in http://www.w3.org/TR/css3-transitions/.

The CSS-WG decided that they should change it to allow the shorthand in the 
future, however.

If you change  -o-transition: margin 0.25s ease-in-out; in the .slide class to 
use the longhand properties, it will work for Opera, e.g. -o-transition: 
margin-left... and margin-right....

It'd be cool if you could fix this. :)


```

Original issue reported on code.google.com by `runningt...@gmail.com` on 5 Oct 2010 at 8:56

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision cd403510d6.
```

Original comment by `ericbide...@html5rocks.com` on 8 Nov 2010 at 2:24
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `ericbide...@html5rocks.com` on 8 Nov 2010 at 2:24
- Added labels: **Priority-Low**, **Milestone-4**, **Slides**
- Removed labels: **Priority-Medium**
